code/source/playfab/PlayFabClientApi.cpp

```diff
@@ -1997,7 +1997,7 @@ namespace PlayFab
         void* customData
     )
     {
-        std::shared_ptr<PlayFabAuthenticationContext> context = request.authenticationContext != nullptr ? request.authenticationContext : PlayFabSettings::staticPlayer;
+        std::shared_ptr<PlayFabAuthenticationContext> context = PlayFabSettings::staticPlayer;
```

Comment:
Ya the non-instance APIs will be removed. Modified as a less disruptive change for now so that the project builds.

---
In reply to: [590863412](https://github.com/jplafonta/XPlatCppSdk/pull/2#discussion_r590863412) [](ancestors = 590863412)